core/rend/CustomTexture.cpp

```diff
@@ -35,6 +35,108 @@
 
 CustomTexture custom_texture;
 
+class CustomTextureSource : public BaseCustomTextureSource
+{
+public:
+	CustomTextureSource(const std::string game_id)
+	{
+		textures_path = hostfs::getTextureLoadPath(game_id);
+		if (!textures_path.empty())
+		{
+			try {
+				hostfs::FileInfo fileInfo = hostfs::storage().getFileInfo(textures_path);
+				if (fileInfo.isDirectory)
+				{
+					NOTICE_LOG(RENDERER, "Found custom textures directory: %s", textures_path.c_str());
+					custom_textures_available = true;
+				}
+			} catch (const FlycastException& e) {
+			}
+		}
+	}
+	bool customTexturesAvailable() override { return custom_textures_available; }
+	bool shouldReplace() const override { return config::CustomTextures; }
+	bool shouldPreload() const override { return config::CustomTextures && config::PreloadCustomTextures; }
+	bool LoadMap() override;
```

Comment:
Naming conventions are a mess in flycast but I'd rather use camel case lowercase for new functions (i.e. `shouldReplace()` rather than `GetTextureCount()` or `LoadMap()`).

Reply:
I must admit I have used all naming conventions already: `c_style_names()`, `CamelCaseUpper()` and `camelCaseLower()` :grin:  but I'm trying to standardize on the latter now.